src/i18n/i18n-loader.js

```diff
@@ -41,4 +41,13 @@ const messages = { // current fallback strategy is to use the first two letters
 
 const getMessages = (locale = getLocale()) => messages[locale];
 
-export { getLocale, getMessages };
+const handleRtl = () => {
+  if (getLocale() === 'ar') {
```

Comment:
Maybe this should be a check on an array of other rtl languages

Reply:
It definitely should once we support more than one language with an RTL dialect. Trying to keep things simple for now.